geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/conf/QueryHints.scala

```diff
@@ -186,6 +187,11 @@ object QueryHints {
       Option(hints.get(LAMBDA_QUERY_PERSISTENT).asInstanceOf[java.lang.Boolean]).forall(_.booleanValue)
     def isLambdaQueryTransient: Boolean =
       Option(hints.get(LAMBDA_QUERY_TRANSIENT).asInstanceOf[java.lang.Boolean]).forall(_.booleanValue)
+    def isScanSeeking: Boolean =
```

Comment:
What's the query hint that someone passes?  Is that documented?

Reply:
it's not documented, i'd consider it an 'internal' hint at the moment. it's `SCAN_SEEKING`

Reply:
Hmm...  under what circumstances would it be worth documenting?  I can both appreciate having "internal" hints and also, if we don't document them, I'd worry that we wouldn't find 'em later when we need 'em.

Reply:
i was mainly thinking this would be useful just for testing and otherwise people would use the sys prop (which is documented).